tests/res_diff_expect/test_write_disagree.ion

```diff
@@ -0,0 +1,44 @@
+$ion_1_0
+{
+ test: {
+  write_compare: {
+   message: "Write_compare: write behavior changed. Description below following the format: 'impl,type,file' where impl is the implementation used for writing files,type is either 'text' or 'binary' and file is the file that be re-writen",
```

Comment:
"re-writen" -> "re-written"

I think it would be good to also mention what the file name means. It's the implementation that generated the write instruction from which the current implementation is re-writing, right?

Reply:
Sure, it's the implementation that is using the 'process' command. I added comment to the message field. (ion_test_driver.py 1107)